CSS changes for better distinction between user name and full name in user list

Jerome-Herbinet opened the pull request:
![2022-08-24_10-39](https://user-images.githubusercontent.com/33763786/186373166-c86f4ccb-22ed-4f15-84b4-e637e614c70d.png)


Signed-off-by: Jérôme Herbinet jerome.herbinet@arawa.fr

Signed-off-by: Jérôme Herbinet <33763786+Jerome-Herbinet@users.noreply.github.com>

Jerome-Herbinet committed:
CSS changes for better visual distinction between user name and full name

Signed-off-by: Jérôme Herbinet jerome.herbinet@arawa.fr

Signed-off-by: Jérôme Herbinet <33763786+Jerome-Herbinet@users.noreply.github.com>

szaimen reviewed (approved):
Lgtm but didn't test

Jerome-Herbinet commented on apps/settings/css/settings.scss:
```diff
@@ -1634,7 +1634,7 @@ doesnotexist:-o-prefocus, .strengthify-wrapper {
 					}
 				}
-
+				
```
I'll do it.

Jerome-Herbinet replied:
@szaimen I solved it, but I did have to add "Signed-off-by: ..." in my commit's description ?

skjnldsv replied:
You have to sign your commits yes @Jerome-Herbinet 
Sign off your commits using `git commit -sm "Your commit message"`

Jerome-Herbinet replied:
Thanks @skjnldsv ;  i've reverted my commit, and i have done a new one ... which is signed.
@szaimen it's OK

Jerome-Herbinet committed:
Fix : Tabs deletion

szaimen commented on apps/settings/css/settings.scss:
```diff
@@ -1649,6 +1649,14 @@ doesnotexist:-o-prefocus, .strengthify-wrapper {
+					color: var(--lt-color-text-very-light);
+					.displayName.subtitle {
+						font-style: normal;
+					}					
```
Here seems to be some whitespace behind the bracket that should be removed

Jerome-Herbinet replied:
Done

szaimen commented on apps/settings/css/settings.scss:
```diff
@@ -1649,6 +1649,14 @@ doesnotexist:-o-prefocus, .strengthify-wrapper {
+					.displayName.subtitle {
+						font-style: normal;
+					}					
+				}				
```
Here as well?

Jerome-Herbinet replied:
Done

Jerome-Herbinet committed:
Revert "Fix : Tabs deletion"

This reverts commit 6879c01c917f055c296d76dc2849d2bf50751c69.

Jerome-Herbinet committed:
Fix : Deleting some tabs (code cleaning)

Signed-off-by: Jérôme Herbinet jerome.herbinet@arawa.fr

Jerome-Herbinet commented:
What is in the commit branch ? is this normal ? any problem ?
[3rdparty](https://github.com/nextcloud/server/pull/33669/files#diff-c16cb0249ed0e5de41d9e31fd3ca629585140a3a21b90b5ddffb467e8463d2a1)
Viewed
Submodule [3rdparty](https://github.com/nextcloud/3rdparty) updated 164 files

jancborchardt reviewed (changes requested):
Good idea! Some design feedback:
- [ ] We never use italic cause it is not so easily legible
- [ ] Instead, we could bold the display name
- [ ] Display name should show in first row, then account name, so it doesn't look weird with a bold subline
- [ ] That variable looks non-standard, we only use either color-main-text or color-text-maxcontrast. But if we bold the display name, we don't need to reduce the color of the account name.

szaimen commented:
> What is in the commit branch ? is this normal ? any problem ? [3rdparty](https://github.com/nextcloud/server/pull/33669/files#diff-c16cb0249ed0e5de41d9e31fd3ca629585140a3a21b90b5ddffb467e8463d2a1) Viewed Submodule [3rdparty](https://github.com/nextcloud/3rdparty) updated 164 files

This change needs to be reverted. See https://stackoverflow.com/questions/30275691/removing-subproject-commit-from-github/30275888#30275888

Jerome-Herbinet committed:
Removing submodule

Jerome-Herbinet commented:
> 

@szaimen I think I've removed it ; can you tell me if it's OK ?

Jerome-Herbinet commented:
> Good idea! Some design feedback:
> 
>     * [ ]  We never use italic cause it is not so easily legible
> 
>     * [ ]  Instead, we could bold the display name
> 
>     * [ ]  Display name should show in first row, then account name, so it doesn't look weird with a bold subline
> 
>     * [ ]  That variable looks non-standard, we only use either color-main-text or color-text-maxcontrast. But if we bold the display name, we don't need to reduce the color of the account name.

@jancborchardt thanks for your suggestions ; I'll take care of it.

Jerome-Herbinet committed:
Revert "Removing submodule"

This reverts commit 3e2190e33c65d55143a73f22ed904cfc8e67a805.

Jerome-Herbinet committed:
Revert "Revert "Removing submodule""

This reverts commit 3f0ca0d20d034ff5f985599216192a5ca399f0cd.

Jerome-Herbinet committed:
Folling jancborchardt requests with some comments

@jancborchardt
Italic removed. Bold "only" is not enough.
Bold AND "color-main-text" is too much (too aggressive).
So I propose only using color-main-text.
I think I don't have skills to invert lines (and I'm afraid of doing this myself now) ; can someone else do it ?

Signed-off-by: Jérôme Herbinet jerome.herbinet@arawa.fr

Jerome-Herbinet commented:
* following

szaimen commented on .gitmodules:
```diff
@@ -1,3 +0,0 @@
-[submodule "3rdparty"]
-	path = 3rdparty
-	url = https://github.com/nextcloud/3rdparty.git
```
hm... seems like following my link removed too much :/

@skjnldsv do you know how to fix that?

skjnldsv replied:
Nope :(

Jerome-Herbinet replied:
@StCyr une idée ?

szaimen commented:
@Jerome-Herbinet thanks! Can you please post a new screenshot? That would be great!

Jerome-Herbinet commented:
> @Jerome-Herbinet thanks! Can you please post a new screenshot? That would be great!

@szaimen here it is :
![2022-08-25_15-29](https://user-images.githubusercontent.com/33763786/186677972-a9baf415-837f-4371-9f65-68c4d4884ca2.png)

szaimen commented:
cc @jancborchardt

jancborchardt commented:
@Jerome-Herbinet almost :) could you:
- Unbold the account name, as the display name is already bold
- Put display name in the first line, accoint in the second (not soo important if difficult)

Jerome-Herbinet commented:
> @Jerome-Herbinet almost :) could you:
> 
>     * Unbold the account name, as the display name is already bold
> 
>     * Put display name in the first line, accoint in the second (not soo important if difficult)

@jancborchardt in fact the text isn't bold ; the other screenshot boldness is certainly due to the default font of my Ubuntu Laptop.

Here is a screenshot of the same NC instance, under Manjaro. 

![scr](https://user-images.githubusercontent.com/33763786/186710584-16ff02b3-83ea-433f-98de-f7b4114d375d.jpg)

So do you want me to bold something ?

And about putting the display name in first position, I won't be able to do it my self (sorry).

jancborchardt commented:
@Jerome-Herbinet ah got it, thanks for the screenshot! Then I'd only say it would be nice to bold the display name (best use a `<strong>` tag for semantic markup).

Jerome-Herbinet committed:
Wider css class call

Remove "displayName" class call to target "subtitle" class in users list AND in table header.
Signed-off-by: Jérôme Herbinet jerome.herbinet@arawa.fr

Jerome-Herbinet committed:
Invert user and display name (+ bold display name)

Signed-off-by: Jérôme Herbinet jerome.herbinet@arawa.fr

Jerome-Herbinet committed:
Header : Invert display (+ bold) and user name

Signed-off-by: Jérôme Herbinet jerome.herbinet@arawa.fr

Jerome-Herbinet commented:
> @Jerome-Herbinet ah got it, thanks for the screenshot! Then I'd only say it would be nice to bold the display name (best use a `\<strong\>` tag for semantic markup).

Done @jancborchardt 
![2022-08-25_15-29](https://user-images.githubusercontent.com/33763786/186854952-21e369d0-c795-4b88-8a5f-dc70b90c3568.png)
Display name is bold ... event if on my "day" computer, difference between bold and not bold texte isn't very visible ... (with "strong") and display name and user name are inverted now.

Jerome-Herbinet commented:
As a newbie as code contributor, I made an error, by committing [5f9962d](https://github.com/Jerome-Herbinet/server/commit/5f9962d427863f635a4d1d2ac93e1a884e3d8808) ... I think I've cancelled it, can you confirm ? (sorry for the mess, I'm learning).

Jerome-Herbinet commented:
I'm concerned about submodule problem ; do you have a tip to solve it ?

Jerome-Herbinet commented:
Hello,
If the Submodule problem is not solvable, do you advise me to close this PR and to create a new one (hopping that it will work better) ?

szaimen commented:
> If the Submodule problem is not solvable, do you advise me to close this PR and to create a new one (hopping that it will work better) ?

Yesy that is probably a good idea. Thanks!

Jerome-Herbinet commented:
> > If the Submodule problem is not solvable, do you advise me to close this PR and to create a new one (hopping that it will work better) ?
> 
> Yesy that is probably a good idea. Thanks!

OK @szaimen, I close it and tell you when the new PR is created (I'll cross the fingers).
